command-snapshot.json

```diff
@@ -1,29 +1,29 @@
 [
   {
-    "command": "force:community:create",
+    "command": "community:create",
     "plugin": "@salesforce/plugin-community",
     "flags": [
-      "apiversion",
+      "api-version",
       "description",
       "json",
       "loglevel",
       "name",
-      "targetusername",
-      "templatename",
-      "urlpathprefix"
+      "target-org",
+      "template-name",
+      "url-path-prefix"
     ],
-    "alias": []
+    "alias": ["force:community:create"]
   },
   {
-    "command": "force:community:publish",
+    "command": "community:list:template",
     "plugin": "@salesforce/plugin-community",
-    "flags": ["apiversion", "json", "loglevel", "name", "targetusername"],
-    "alias": []
+    "flags": ["api-version", "json", "loglevel", "target-org"],
+    "alias": ["force:community:template:list", "community:template:list"]
```

Comment:
is the `community:template:list` alias required? Tax-free should take care of that right?